packages/flutter/test/widgets/heroes_test.dart

```diff
@@ -1925,6 +1925,90 @@ Future<void> main() async {
     expect(tester.takeException(), isAssertionError);
   });
 
+  testWidgets('Can push/pop on outer Navigator if nested Navigators contains same Heroes', (WidgetTester tester) async {
```

Comment:
I might miss something, but is this really related to nested navigators? I thought the problem is the CupertinoTabScaffold builds a stack where some of the children are offstage and the heroes in those children conflicting with each other?

Reply:
same here

Reply:
This is not addressed yet?

Reply:
Heroes are animated by screen transition. This is related to Navigator. The problem is caused by multiple nested Navigators.